simapp/app.go

```diff
@@ -274,6 +274,18 @@ func (app *SimApp) ModuleAccountAddrs() map[string]bool {
 	return modAccAddrs
 }
 
+// BlacklistedAccAddrs returns all the app's module account addresses black listed for receiving tokens.
+func (app *SimApp) BlacklistedAccAddrs() map[string]bool {
```

Comment:
To make this more flexible, I instead recommend having a map of `map[string]bool` that only includes `distr.ModuleName` atm and check if `acc` is in that. This is opposed to checking directly against `distr.ModuleName` 👍

Reply:
Implemented